fdbserver/workloads/BulkLoading.actor.cpp

```diff
@@ -170,12 +173,20 @@ struct BulkLoading : TestWorkload {
 						if (bulkLoadTaskState.getRange() != KeyRangeRef(res[i].key, res[i + 1].key)) {
 							continue; // Ignore outdated task
 						}
-						if (bulkLoadTaskState.phase != BulkLoadPhase::Complete) {
+						if (bulkLoadTaskState.phase != BulkLoadPhase::Complete &&
+						    bulkLoadTaskState.phase != BulkLoadPhase::Error) {
 							TraceEvent("BulkLoadingWorkLoadIncompleteTasks")
 							    .setMaxEventLength(-1)
 							    .setMaxFieldLength(-1)
 							    .detail("Task", bulkLoadTaskState.toString());
-							return false;
+							return std::make_pair(errorTasks, false);
+						}
+						if (bulkLoadTaskState.phase == BulkLoadPhase::Error) {
+							TraceEvent("BulkLoadingWorkLoadFailedTasks")
+							    .setMaxEventLength(-1)
+							    .setMaxFieldLength(-1)
+							    .detail("Task", bulkLoadTaskState.toString());
+							errorTasks.push_back(bulkLoadTaskState);
 						}
```

Comment:
Is it ok if we get to here w/o tripping any of the above checks? E.g. phase is complete and not an error?

Reply:
If a task is complete, the while loop just keep going. If all tasks complete, the while loop ends and return "true" state at the end of the actor.